packages/nodes-base/nodes/Stellar/actions/entities/ILiquidtyPoolPrice.ts

```diff
@@ -0,0 +1,4 @@
+export interface ILiquidtyPoolPrice {
+	n: string;
+	d: string;
```

Comment:
What does this mean?

Reply:
This interface is not used. I will remove it.